scripts/build.sh

```diff
@@ -11,3 +15,4 @@ mv terminus.phar $TERMINUS_BIN_DIR/terminus.phar
 cp $TERMINUS_BIN_DIR/terminus.phar $TERMINUS_BIN_DIR/terminus
 chmod +x $TERMINUS_BIN_DIR/terminus.phar
 chmod +x $TERMINUS_BIN_DIR/terminus
+php $TERMINUS_BIN_DIR/composer.phar update
```

Comment:
Probably doesn't matter, but maybe cleanup so someone doesn't wonder why it's in there.